cassandra/policies.py

```diff
@@ -528,10 +528,14 @@ class ExponentialReconnectionPolicy(ReconnectionPolicy):
     a set maximum delay.
     """
 
-    def __init__(self, base_delay, max_delay):
+    def __init__(self, base_delay, max_delay, max_attempts=None):
```

Comment:
I think we need to keep the default 64 to match the previous behavior if the parameter is unspecified.

Does it ever make sense to try forever (None)? With exponential backoff, the retry interval gets very large, and might lead to situations where reconnects are not very responsive (with a retry scheduled, but *way* in the future).

As an aside, are you trying to solve a specific problem here?